xcode/Subconscious/Shared/Components/Settings/GatewayURLSettingsView.swift

```diff
@@ -88,6 +117,129 @@ struct GatewayProvisionLabel: View {
     }
 }
 
+// MARK: Provisioning
+
+struct ValidatedInviteCodeFormField: View {
```

Comment:
We could consider breaking these section views out into separate files. Non-blocking.

Reply:
Done!